bento_map/models.py

```diff
@@ -124,14 +122,51 @@ def element(self):
             "id": self.name,
             "name": self.name,
             "provider": DATABASE_PROVIDER,
-            "timestamp": self.created_at_ts,
+            "timestamp": self.timestamp,
             "to": "{}/{}_{}".format(
                 HOST_COLLECTION, MAP_PROVIDER, self.identifier
             )
         }
 
     @property
     def key(self):
-        return "{}/{}_{}".format(
-            DB_HOST_COLLECTION, DATABASE_PROVIDER, self.name
-        )
+        return "{}_{}".format(DATABASE_PROVIDER, self.name)
+
+
+class Tsuru(BaseModel):
+
+    def __init__(self, infra_name, db_name, timestamp, environment):
+        super(Tsuru, self).__init__(infra_name, db_name, timestamp)
+        self.name = db_name
+        self.environment = environment
+
+    @property
+    def collection(self):
+        return DB_TSURU_COLLECTION
+
+    @property
+    def type(self):
+        return "edges"
+
+    @property
+    def element(self):
+        provider = self.environment.replace(TSURU_PRODUCTION_ENVIRONMENT, '')
+        if provider:
+            provider = '-' + provider
+        provider = 'tsuru_tsuru-dbaas' + provider
```

Comment:
tsuru_tsuru-dbaas is hardcoded ?

Reply:
I will put that in a environment variable, thanks!